summary_workflows/quantification/APAeval_Q_test_data/metrics_ref_datasets/test.gt.mm10_test.bed

```diff
@@ -0,0 +1,14 @@
+chr1	1000000	1000001	same_pd_21	20	+
```

Comment:
should it be this file: `summary_workflows/quantification/APAeval_Q_test_data/metrics_ref_datasets/test.mm10_test.gt.bed`? 
If this is a new one and should be used in the test, please change it in `nextflow.config` to use it, see my other comment.